docs/ocl/overview.rst

```diff
@@ -1,4 +1,10 @@
 Overview
 ========
 
-TODO
\ No newline at end of file
+`https://www.orocos.org/ocl <https://www.orocos.org/ocl>`_
```

Comment:
_Note:_ https://www.orocos.org/ocl (like https://www.orocos.org/rtt) point to the outdated 1.0 version from before 2010. There have been many more components in this version of OCL, which also justify the name of the package as a library of commonly used components (cf. https://github.com/orocos-toolchain/ocl/tree/ocl-1.0-svn). So the page above is actually outdated.

So it's okay to reference the old page here, but better to copy over and update the text snippets as orocos.org will be replaced by a simple landing page soon.

Since 2.0 OCL became more like a collection of libraries and components with optional extensions of RTT itself, like the components below. There are no components anymore that have a real functional value, like for motion control or hardware drivers. I am not sure why it is like that. Most likely these components just never have been ported to 2.0.

In its current stage I would propose to merge what remains of OCL into subdirectories of the RTT repository at some point, with the option to disable them at compile time.